pkg/machine/qemu/machine.go

```diff
@@ -647,6 +648,35 @@ func getDevNullFiles() (*os.File, *os.File, error) {
 	return dnr, dnw, nil
 }
 
+// qemuProcess returns the os.Process of the running QEMU instance.
+// May be nil.
+func (v *MachineVM) qemuProcessRunning() (bool, int, error) {
```

Comment:
instead of return both a bool and int would it not be simpler to only use the int and the caller knows int == -1 means not running

Reply:
I was considering it but preferred the expressiveness oft a bool. `if qemuRunning` seems easier to understand than `qemuPid != -1`.